CMakeLists.txt

```diff
@@ -379,6 +379,9 @@ cmake_dependent_option(IREE_TARGET_BACKEND_LLVM_CPU_WASM "Enables WebAssembly in
 cmake_dependent_option(IREE_TARGET_BACKEND_METAL_SPIRV "Enables the 'metal-spirv' compiler target backend" ${IREE_TARGET_BACKEND_DEFAULTS} ${IREE_BUILD_COMPILER} OFF)
 cmake_dependent_option(IREE_TARGET_BACKEND_VULKAN_SPIRV "Enables the 'vulkan-spirv' compiler target backend" ${IREE_TARGET_BACKEND_DEFAULTS} ${IREE_BUILD_COMPILER} OFF)
 
+# Default target backends that are not yet fully supported but are being brought up.
+cmake_dependent_option(IREE_TARGET_BACKEND_ROCM "Enables the 'rocm' compiler target backend" ON ${IREE_BUILD_COMPILER} OFF)
```

Comment:
Docs will be a little out of date with this change

https://www.iree.dev/guides/deployment-configurations/gpu-cuda-rocm/#get-the-iree-compiler

![image](https://github.com/openxla/iree/assets/4010439/3ab05f77-a23f-4d20-8c3c-75ffd35aff39)

Source file here: https://github.com/openxla/iree/blob/main/docs/website/docs/guides/deployment-configurations/gpu-cuda-rocm.md

---

We could update the docs here or maybe wait for a Python release that includes the code before changing the docs. This could also be a reason to wait a bit before promoting a stable release (so the next stable includes this), https://groups.google.com/g/iree-discuss/c/_cBAm5VubN4/m/0-v0t9qvAwAJ

Reply:
None of this is *supported* yet. We are enabling by default in order to work towards supporting it. I think we leave the docs as-is for now.